shell/platform/darwin/ios/framework/Source/FlutterViewController.mm

```diff
@@ -164,7 +165,6 @@ - (void)performCommonViewControllerInitialization {
 
   [self setupNotificationCenterObservers];
 
-  [self connectToEngineAndLoad];
```

Comment:
Will moving the call to fire up the engine until `viewWillAppear` delay the first frame rendered by the Flutter framework?